.changes/windows-0.37.0.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Update the windows-rs crate to the latest 0.37.0 release. This depends on rustc version 1.61 for some `const` generic support which was just stabilized, so on Windows the MSRV is effectively 1.61 now.
```

Comment:
```suggestion
Update the windows-rs crate to the latest 0.37.0 release. This depends on rustc version 1.61 for some `const` generic support which was just stabilized, so on Windows the MSRV is effectively 1.61 now.
```
Can you also update thr MSRV in `Cargo.toml`?

Reply:
Sure, I figured it didn't need to be raised for other platforms since they won't include the `windows-rs` dependency, but I can see how keeping track of different MSRVs on each platform would be riskier.

I was also thinking that the MSRV in `webview2-com` would take care of enforcement, but that's only applicable on `wry`, and `tao` does use the `windows-implement` macro for some COM interfaces.

Reply:
I saw some Rust maintainers complaining about that too, because the Debian package is supposed to be a build dependency for other packages, not used for developers since it locks you to an old version. I think not having the same MSRV across all platforms is tricky since most users cannot see the build error as an outdated Rust version error.

Reply:
Just noticed the new comments, is the consensus that I should revert that change to the MSRV for `tao` and `wry`?

Windows developers are probably more likely to use `rustup` since the OS doesn't come with redistributable toolchain packages (unless you count cross-compilation), so I think it's fair to require them to perform an update. Unfortunately, none of the Windows dependencies in `tao` specify an MSRV, so if they try to build an app with only `tao`, the errors they get will probably be about using unstable features instead of needing to run the upgrade. I'll try to merge a PR to add an MSRV to `windows-implement` since that specific crate is using those features (although you also get that MSRV dependency if you include the `implement` feature in the base `windows` crate), and `tao` depends on that crate so it'll start getting the MSRV errors after that.

The `wry` MSRV should be fine if it's just unspecified. It depends on `webview2-com` (and `windows-implement`, plus the `implement` feature in `windows`), and `webview2-com` specifies the MSRV.

Reply:
Here's the PR I started for `windows-implement`: https://github.com/microsoft/windows-rs/pull/1773.